docker/ceph/start-ceph.sh

```diff
@@ -53,7 +53,9 @@ echo 'vstart.sh completed!'
 "$CEPH_BIN"/ceph osd pool application enable rbd-pool rbd
 
 # Configure Object Gateway:
-/docker/set-rgw.sh
+if [ "$RGW" -gt 0 ]; then
```

Comment:
```suggestion
if [ "$RGW" -gt 0 || "$RGW_MULTISITE" == 1 ]; then
```

Reply:
why's that? I assume that for using RGW_MULTISITE you have first enabled RGW, no?

Reply:
Because this:
https://github.com/rhcs-dashboard/ceph-dev/blob/master/docker/ceph/set-start-env.sh#L19

By setting RGW=0 when multisite we ensure that in vstart.sh no default zonegroup and zone are created in first place.

Reply:
Ok, understood! Thanks for the clarification!